file/file_prefetch_buffer.cc

```diff
@@ -76,6 +76,8 @@ void FilePrefetchBuffer::CalculateOffsetAndLen(size_t alignment,
     bufs_[index].buffer_.AllocateNewBuffer(
         static_cast<size_t>(roundup_len), copy_data_to_new_buffer,
         chunk_offset_in_buffer, static_cast<size_t>(chunk_len));
+  } else {
```

Comment:
Is there any reason why are we assigning 0 size?

Reply:
The other branches all make function calls that result in `bufs_[index].buffer_.CurrentSize() == chunk_len`. I thought that makes sense because then `bufs_[index].buffer_.Destination()` returns the correct position to append to. This branch was the only exception, and chunk_len is zero here, so I set `CurrentSize()` to zero.